src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java

```diff
@@ -79,7 +79,7 @@
     private final String paymentReference;
     private final DynamicList applicantSolicitor1PbaAccounts;
     private final ClaimType claimType;
-    private final SuperClaimType superClaimType;
```

Comment:
previously SuperClaimType was set as UNSPEC in the CaseDataBuilder's build() method. I wasn't able to write test cases for SPEC because of this. (Please check DefendantResponseApplicantNotificationHandlerTest.java line 102)
So making SuperClaimType not final and setting SuperClaimType as UNSPEC in atStateClaimDraft()and later setting it as a SPEC claim in the test case solved the issue.